src/Corax/Queries/UnaryMatch.Ordinal.cs

```diff
@@ -947,7 +947,7 @@ private static int FillFuncNumericalAll<TComparer>(ref UnaryMatch<TInner, TValue
                 return new UnaryMatch<TInner, TValueType>(
                     in inner, UnaryMatchOperation.GreaterThan,
                     searcher, fieldId, value,
-                    mode == UnaryMatchOperationMode.Any ? &FillFuncNumericalAny<GreaterThanMatchComparer> : &FillFuncNumericalAny<GreaterThanMatchComparer>, &AndWith,
+                    mode == UnaryMatchOperationMode.Any ? &FillFuncNumericalAny<GreaterThanMatchComparer> : &FillFuncSequenceAll<GreaterThanMatchComparer>, &AndWith,
```

Comment:
Pretty sure that this is a valid fix, but I want another pair of eyes here.

Reply:
This is the builder for numerics. Notice the above condition when we check if `TValueType` is a `Slice`.

Reply:
I think so

Reply:
Yes.